src/index.html

```diff
@@ -5,8 +5,327 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <nav class="header__nav">
+        <a href="" class="header__logo">
+          <img src="images/logo.png" alt="MOYO">
+        </a>
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="" class="header__link header__link--is-active is-active">Apple</a>
+          </li>
+          <li>
+            <a href="" class="header__link">Samsung</a>
+          </li>
+          <li>
+            <a href="" class="header__link">смартфоны и телефоны</a>
+          </li>
+          <li>
+            <a
+              href=""
+              data-qa="hover"
+              class="header__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li>
+            <a href="" class="header__link">гаджеты</a>
+          </li>
+          <li>
+            <a href="" class="header__link">планшеты</a>
+          </li>
+          <li>
+            <a href="" class="header__link">фото</a>
+          </li>
+          <li>
+            <a href="" class="header__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-catalog">
+      <div class="card main-catalog__card" data-qa="card">
+        <div class="card__picture card__picture--margin"></div>
```

Comment:
Please, read this message 
https://mateacademy.slack.com/archives/C0265RETQL9/p1626365372088200